glusterfs/datadog_checks/glusterfs/check.py

```diff
@@ -0,0 +1,207 @@
+    'size_free': 'size.free',
+    'size_total': 'size.total',
+    'inodes_used': 'inodes.used',
+    'inodes_total': 'inodes.total',
+    'online': 'online',
+}
+
+VOL_SUBVOL_STATS = {
+    'disperse': 'disperse',
+    'disperse_redundancy': 'disperse_redundancy',
+    'replica': 'replica',
+}
+
+VOLUME_STATS = {
+    'v_used_percent': 'used.percent',
+    'num_bricks': 'bricks.count',
+    'distribute': 'distribute',
+    'v_size_used': 'size.used',
+    'v_size': 'size.total',
+    'snapshot_count': 'snapshot.count',
+}
+
+VOLUME_STATS.update(GENERAL_STATS)
+VOLUME_STATS.update(VOL_SUBVOL_STATS)
+
+BRICK_STATS = {'block_size': 'block_size'}
+BRICK_STATS.update(GENERAL_STATS)
+
+PARSE_METRICS = ['v_size_used', 'v_size']
+
+GLUSTER_VERSION = 'glfs_version'
+CLUSTER_STATUS = 'cluster_status'
+
+GSTATUS_PATH = '/opt/datadog-agent/embedded/sbin/gstatus'
+INSTALL_PATH = '/usr/local/bin/gstatus'
+
+
+class GlusterfsCheck(AgentCheck):
+    __NAMESPACE__ = 'glusterfs'
+
+    CLUSTER_SC = "cluster.health"
+    VOLUME_SC = "volume.health"
+    BRICK_SC = "brick.health"
+
+    def __init__(self, name, init_config, instances):
+        # type: (*Any, **Any) -> None
+        super(GlusterfsCheck, self).__init__(name, init_config, instances)
+        self._tags = self.instance.get('tags', [])
+
+        # Check if customer set gstatus path
+        if init_config.get('gstatus_path'):
+            self.gstatus_cmd = init_config.get('gstatus_path')
+        else:
+            if os.path.exists(GSTATUS_PATH):
+                self.gstatus_cmd = GSTATUS_PATH
+            elif os.path.exists(INSTALL_PATH):
+                self.gstatus_cmd = INSTALL_PATH
+            else:
+                raise ConfigurationError(
+                    'Glusterfs check requires `gstatus` to be installed or set the path to the installed version.'
+                )
+        self.log.debug("Using gstatus path `%s`", self.gstatus_cmd)
+
+    def check(self, _):
+        use_sudo = _is_affirmative(self.instance.get('use_sudo', False))
+        if use_sudo:
+            test_sudo = os.system('setsid sudo -l < /dev/null')
+            if test_sudo != 0:
+                raise Exception('The dd-agent user does not have sudo access')
+            gluster_args = ['sudo', self.gstatus_cmd]
+        else:
+            gluster_args = [self.gstatus_cmd]
+
+        # Ensures units are universally the same by specifying the --units flag
+        gluster_args += ['-a', '-o', 'json', '-u', 'g']
+        self.log.debug("gstatus command: %s", gluster_args)
+        output, _, _ = get_subprocess_output(gluster_args, self.log)
+        gstatus = json.loads(output)
+
+        if 'data' in gstatus:
+            data = gstatus['data']
+            self.submit_metric(data, 'cluster', CLUSTER_STATS, self._tags)
+
+            self.submit_version_metadata(data)
+
+            if 'volume_summary' in output:
+                self.parse_volume_summary(data['volume_summary'])
```

Comment:
data or output? You're using both
Also we can use a constant for this, especially because you're using one for `CLUSTER_STATUS`